unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -4593,41 +4585,44 @@ public void PickupContainedObjects(SimObjPhysics target)
             }
         }
 
-        public void DropContainedObjects(SimObjPhysics target) {
+        public void DropContainedObjects(
```

Comment:
I know you didn't write this method originally, but could it be made protected or package level visibility instead of public?

Reply:
It seems to be used within the `InstatiatePrefabTest` class so I believe it needs to be public.